base/healthcheck-docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM ubuntu:19.10
+
+RUN apt-get update && apt-get upgrade -y
+
+RUN /usr/sbin/useradd --no-create-home -u 1000 user
+
+RUN apt-get -y install python2.7 python-pip
+RUN pip install pwntools
+RUN mkdir -p /home/user/.pwntools-cache && echo never > /home/user/.pwntools-cache/update
```

Comment:
why don't we cache this? it's the slowest part probably.

we could have this with the other kctf-nsjail-* images.

Reply:
hmm, I would need to know where the files get installed by pip. Maybe I can use a virtualenv, wdyt?

Reply:
yea virtualenv would be awesome :)

Reply:
done